app/controllers/wikis_controller.rb

```diff
@@ -0,0 +1,65 @@
+class WikisController < ApplicationController
+  before_action :authenticate_user!, except: [:show, :index]
+
+  def new
+    @wiki = Wiki.new
+  end
+
+  def index
+    @wikis = Wiki.all.order(created_at: :desc)
+  end
+
+  def create
+    @wiki = current_user.wiki.build(wiki_params)
```

Comment:
Do you need the next line since you're building the wiki off of the current_user object?

Reply:
Also, I believe you'll want this to be `current_user.wikis.build`.